source/hands_on/ppo.rst

```diff
@@ -90,9 +90,111 @@ The policy gradient and value update of PPO is implemented as follows:
+|                     |                 |                                                     |github.com/opendilab/     |                      |
+|                     |                 |                                                     |DI-engine/tree/main/dizoo/|   Tianshou(2700)     |
+|Hopper               |  3000           |.. image:: images/benchmark/hopper_onppo.png         |mujoco/config/serial/     |      Sb3(1567)       |
+|                     |                 |                                                     |hopper/hopper_onppo_config|                      |
+|(Hopper-v3)          |                 |                                                     |.py>`_                    |                      |
++---------------------+-----------------+-----------------------------------------------------+--------------------------+----------------------+
+|                     |                 |                                                     |`config_link_w <https://  |                      |
+|                     |                 |                                                     |github.com/opendilab/     |                      |
+|Walker2d             |                 |                                                     |DI-engine/tree/main/dizoo/|   Tianshou(4500)     |
+|                     |  3000           |.. image:: images/benchmark/walker2d_onppo.png       |mujoco/config/serial/     |     Sb3(1230)        |
+|(Walker2d-v3)        |                 |                                                     |walker2d/walker2d_        |                      |
+|                     |                 |                                                     |onppo_config.py>`_        |                      |
++---------------------+-----------------+-----------------------------------------------------+--------------------------+----------------------+
+|                     |                 |                                                     |`config_link_ha <https:// |                      |
+|                     |                 |                                                     |github.com/opendilab/     |                      |
+|Halfcheetah          |                 |                                                     |DI-engine/tree/main/dizoo/|   Tianshou(7194)     |
+|                     |  3500           |.. image:: images/benchmark/halfcheetah_onppo.png    |mujoco/config/serial/     |     Sb3(1976)        |
+|(Halfcheetah-v3)     |                 |                                                     |halfcheetah/halfcheetah   |                      |
+|                     |                 |                                                     |_onppo_config.py>`_       |                      |
++---------------------+-----------------+-----------------------------------------------------+--------------------------+----------------------+
+
```

Comment:
Total commets:
In line 26, L^{C L I P}(\theta)=\hat{E}_{t}\left[\min \left(r_{t}(\theta) \hat{A}_{t}, \operatorname{clip}\left(r_{t}(\theta), 1-\epsilon, 1+\epsilon\right) \hat{A}_{t}\right)\right] will not report error in LaTex